package.json

```diff
@@ -10,7 +10,7 @@
 	"keywords": [],
 	"author": "",
 	"devDependencies": {
-		"shared": "file:./shared",
-		"@changesets/cli": "^2.26.2"
-	}
+		"@biomejs/shared": "workspace:^"
+	},
+	"packageManager": "pnpm@8.15.4"
```

Comment:
Is there a reason why we don't append the shasum to this value? By default core pack should have added it?

Reply:
i'm used to removing it in my small personal projects and also saw some big repos that don't use it, but yeah its a good idea to have it, will add